src/Backend/SgateShopgatePlugin/Components/Favorites.php

```diff
@@ -399,4 +418,23 @@ private function removeProductFromWishList($articles, $userId)
 
         return true;
     }
+
+    /**
+     * check for product exists in active category
+     *
+     * @param $productId
```

Comment:
type missing in phpdoc + please capitalize the description text of the phpdoc

Reply:
Good point, the wording is a little off too.